table/server/common/src/test/java/alluxio/table/common/udb/PathTranslatorTest.java

```diff
@@ -27,6 +31,13 @@
   @Rule
   public ExpectedException mException = ExpectedException.none();
 
+  @Rule
+  public ConfigurationRule mConfiguration =
+      new ConfigurationRule(
+          ImmutableMap.of(PropertyKey.MASTER_HOSTNAME, "master",
+            PropertyKey.MASTER_RPC_PORT, "19998"),
```

Comment:
Let's define two string constant `MASTER_HOSTNAME` and  `MASTER_RPC_PORT`
then replace the literal in the following and also in the test cases with the constant.
This is a better practise for writing tests -- more maintainable and more flexible

```suggestion
          ImmutableMap.of(PropertyKey.MASTER_HOSTNAME,  MASTER_HOSTNAME,
            PropertyKey.MASTER_RPC_PORT, MASTER_RPC_PORT),
```
Also, please define `MASTER_RPC_PORT` to be non-default value.